tests/EvoSC.Common.Tests/Util/MatchSettings/MatchSettingsXmlParserTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EvoSC.Common.Util.MatchSettings;
+using Xunit;
+
+namespace EvoSC.Common.Tests.Util.MatchSettings;
+
+public class MatchSettingsXmlParserTests
+{
+    [Fact]
+    public async Task Parses_MatchSettings_With_All_Sections()
+    {
+        var xml = """
```

Comment:
A potential solution to having the XMLs as strings within the test is to add a "resources" folder under EvoSC.Common.Tests which contains all XML's used for testing purposes.  One of the benefits by doing it this way, is that it's easy for new contributors to locate all XMLs and JSONs that are used within the tests, which again can be used to get a better understanding of what the different XML/JSONs need to look like in order to incorporate them into their code.

Reply:
In Skatteetaten we would also give the XMLs names similar to their purpose, e.g. `matchsettings_with_all_sections.xml` `matchsettings_missing_attributes.xml`